scylla/tests/ccm_integration/ccm/mod.rs

```diff
@@ -1,8 +1,27 @@
 pub(crate) mod cluster;
 mod logged_cmd;
 pub(crate) mod node_config;
+
+use std::future::Future;
+use std::path::PathBuf;
+use std::sync::Arc;
 use std::sync::LazyLock;
+
+use cluster::Cluster;
+use cluster::ClusterOptions;
+use tokio::sync::Mutex;
 use tracing::info;
+
+pub(crate) static CLUSTER_VERSION: LazyLock<String> =
+    LazyLock::new(|| std::env::var("SCYLLA_TEST_CLUSTER").unwrap_or("release:6.2.2".to_string()));
```

Comment:
:wrench: Let's extract `DEFAULT_SCYLLADB_RELEASE` = `release:6.2.2`, for easier updates in the future.